.github/workflows/android-ci.yml

```
@@ -0,0 +1,74 @@
+          key: gradle-${{ runner.os }}-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          # 정확한 캐시 매치가 없을 때 부분 캐시 사용
+          restore-keys: |
+            gradle-${{ runner.os }}-
+
+      - name: Run detekt
+        run: ./gradlew detekt
+
+      - name: Run Android Lint
+        run: ./gradlew lintDebug
+
+      # 테스트 코드 작성 후 활성화
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)